dnsdist out-of-tree build issues

- Program: dnsdist
 - Issue type: Bug report

### Short description
dnsdist out of tree builds fail while leaving garbage behind

### Environment
OpenBSD current (not really relevant), dnsdist 1.1.0

### Steps to reproduce
1. `mkdir build && cd build`
2. `../configure --flags`
3. `make`

### Expected behaviour
A working build.

### Actual behaviour
```
./incfiles > htmlfiles.h
/bin/sh: ./incfiles: not found
*** Error 127 in target 'htmlfiles.h', line 1736 of Makefile (continuing)
Build for all aborted
`all' not remade because of errors.
```

### Other information
If I run `make` again, it does continue, because the above has left behind a zero byte htmlfiles.h which satisfies `make`'s checks.

```
In file included from ../ext/yahttp/yahttp/yahttp.hpp:12:0,
                 from ../dnsdist-web.cc:29:
../ext/yahttp/yahttp/yahttp-config.h:1:29: fatal error: ../../../config.h: No such file or directory
 #include "../../../config.h"
                             ^
compilation terminated.
gmake[2]: *** [Makefile:1006: dnsdist-web.o] Error 1
gmake[2]: Leaving directory '/root/dnsdist-0.0.1367gacd402e/build-O0'
gmake[1]: *** [Makefile:1133: all-recursive] Error 1
gmake[1]: Leaving directory '/root/dnsdist-0.0.1367gacd402e/build-O0'
gmake: *** [Makefile:829: all] Error 2
```

--- a/ext/yahttp/yahttp/yahttp-config.h
+++ b/ext/yahttp/yahttp/yahttp-config.h
@@ -1,1 +1,1 @@
-#include "../../../config.h"
+#include "config.h"

